localstripe/server.py

```diff
@@ -286,6 +288,8 @@ def localstripe_js(request):
 
 app.router.add_get('/js.stripe.com/v3/', localstripe_js)
 
+async def healthcheck(request):
+    return web.json_response('pong')
```

Comment:
thinking outloud, iirc since it's a json response of a string, if we care about the value, we just have to make sure to check the value `'"pong"'` with quotes